src/components/search/SearchError.jsx

```diff
@@ -10,26 +10,24 @@ const SearchError = ({ title }) => {
   const renderDialog = useCallback(
     () => (
       <div className="lead d-flex align-items-center py-3">
-        <div className="mr-3">
-          <FontAwesomeIcon icon={faExclamationTriangle} size="2x" />
-        </div>
-        <div>
+        <Alert.Heading>
           {searchErrorMessage.messageTitle}
-          <br />
-          {searchErrorMessage.messageContent}
-        </div>
+        </Alert.Heading>
+        {searchErrorMessage.messageContent}
       </div>
     ),
     [],
   );
 
   return (
-    <StatusAlert
-      alertType="danger"
-      dialog={renderDialog()}
+    <Alert
+      variant="danger"
       dismissible={false}
+      icon={() => <FontAwesomeIcon icon={faExclamationTriangle} size="2x" />}
       open
-    />
+    >
+      {renderDialog()}
```

Comment:
There isn't really much value to having `renderDialog` as a function versus just rendering its children here now instead. Simplifies the implementation by removing a `useCallback`.

Reply:
done